docs: clarify that saml is not available on hobby tier (#140)
siderolabs/docs: public/omni/security-and-authentication/using-saml-with-omni/overview.mdx

---

Omni can integrate with your enterprise SAML provider for authentication and identity management. See also information about [how SAML impacts Omni authentication.](../../security-and-authentication/authentication-and-authorization) Please contact support@siderolabs.com or your account manager to enable SAML on the SaaS version of Omni (N.B. this is only available on Edge tier and above).

> If you self-host Omni for non-production use, you are free to configure SAML yourself.

Reviewer: Makes it clearer that the *use* is the important thing.
```suggestion
If use Omni for non-production, you can self-host and configure SAML yourself.
```
plus maybe it should link to the self-hosting docs?
